guide/payments/coin-selection.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: guide
+title: Coin Selection
+nav_order: 1
+parent: Payments
+permalink: /guide/payments/coin-selection/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/glossary/coin-selection/header.jpg"
+   retina = "/assets/images/guide/glossary/coin-selection/header@2x.jpg"
+   mobile = "/assets/images/guide/glossary/coin-selection/header-mobile.jpg"
+   mobileRetina = "/assets/images/guide/glossary/coin-selection/header-mobile@2x.jpg"
+   alt-text = "Coin Selection"
+   width = 1600
+   height = 800
+   layout = "full-width"
+%}
+
+# Coin selection
+
+Coin selection is the process of choosing which [UTXOs](https://hackmd.io/glossary#utxo) (or “coins”) to fund a bitcoin transaction with (inputs[^1]) when making an on-chain payment. Since your wallet contains multiple coins to arrive at a balance, these are retrieved by an algorithm to fulfill the payment amount for outgoing transactions.
+
+Coin selection, whether an automated or manual process, is important when understanding how we can successfully optimise our bitcoin payments for either cost, speed, or privacy.
+
+There are two categories of coin selection strategies that are used in bitcoin applications:
+
+1. **Automatic Coin Selection** (wallet is delegated to control coin selection on behalf on user)
+2. **Manual Coin Selection** (user controls coin selection)
+
+{% include picture.html
+   image = "/assets/images/guide/glossary/coin-selection/funding-tx.jpg"
+   retina = "/assets/images/guide/glossary/coin-selection/funding-tx@2x.jpg"
+   mobile = "/assets/images/guide/glossary/coin-selection/funding-tx-mobile.jpg"
+   mobileRetina = "/assets/images/guide/glossary/coin-selection/funding-tx-mobile@2x.jpg"
+   alt-text = "Funding TX"
+   width = 800
+   height = 400
+%}
```

Comment:
Nice image but it's too tiny, here's how it looks on my desktop.

![Screenshot 2021-05-27 at 18 20 12](https://user-images.githubusercontent.com/36959754/119862018-57e9ea00-bf18-11eb-9c76-b8181dd0c243.png)